local-storage-service/server/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+verapdf.files.min-space-threshold=5GB
+verapdf.files.base-dir=${FILE_STORAGE_DIR_PATH}
+
+management.endpoints.web.base-path=/status
+server.servlet.context-path=/api
```

Comment:
Let's migrate to `yml` structure and update values as following:
```
server:
  servlet:
    context-path: /api/file
management:
  endpoints:
    web:
      base-path: /status

verapdf:
  files:
    base-dir: ${FILE_STORAGE_DIR_PATH}
    min-space-threshold: 5GB
```

Note that we first put generic Spring props and after that app specific